src/components/ThreejsEditorModal.jsx

```diff
@@ -104,12 +103,17 @@ export class ThreejsEditorModal extends ModalDialog {
         this.domElement.appendChild(sidebar.dom);
         const toolbar = new Toolbar(this.editor);
         this.domElement.appendChild(toolbar.dom);
-
+        const script = new Script(this.editor);
+        this.domElement.appendChild(script.dom);
         this.initializeControls();
-
+        this.cacheSceneFiles();
         this.initializeLights();
     }
 
+    cacheSceneFiles() {
+        THREE.Cache.add("examples/rotation.app.json", JSON.stringify(rotation));
```

Comment:
1. Should we store this in a general `cache.json` file - what about any other examples in the future?
2. A function should have a docstring.